app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.players.player import Player
+from typing import List
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
```

Comment:
Add a blank line between different import groups and order them correctly to follow PEP-8